live-build/config/hooks/configuration/10-disable-pipeline.chroot_early

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -eux
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# This setting is necessary to prevent the "Undetermined Error" issue when
+# downloading packages using apt during the build. chroot_early hooks
+# are run after debootstrap and before packages are installed in lb_chroot.
```

Comment:
> chroot_early hooks are run after debootstrap

Is it possible for us to run into this issue while running `debootsrap`?

Reply:
It doesn't seem so. I've traced debootstrap to see if `apt` or `apt-get` was being called 
but it is not.

Looking at the source code for debootstrap, it seems to read the package manifests manually and is using `wget` directly to fetch the packages.

Reply:
Thanks for confirming!